lighthouse-core/test/index-test.js

```diff
@@ -132,6 +132,31 @@ describe('Module Tests', function() {
     });
   });
 
+  it('should specify the channel as node by default', function() {
```

Comment:
not a big deal, but if making one last commit, could you make these async/await? no need to stick to the style of the file for that :)